pkgs/tools/misc/slop/default.nix

```diff
@@ -14,17 +25,28 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-LdBQxw8K8WWSfm4E2QpK4GYTuYvI+FX5gLOouVFSU/U=";
   };
 
-  nativeBuildInputs = [ cmake pkg-config ];
-  buildInputs = [ glew glm libGLU libGL libX11 libXext libXrender icu ]
-                ++ lib.optional doCheck cppcheck;
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
 
-  doCheck = false;
+  buildInputs = [
+    glew
+    glm
+    libGLU
+    libGL
+    libX11
+    libXext
+    libXrender
+    icu
+    libSM
+  ];
```

Comment:
`cppcheck` was removed from the inputs, it should be explained in the commit message why.

Reply:
I removed it because the build went totally fine without it, does that seem right?